packages/2d/src/lib/code/CodeRange.ts

```diff
@@ -168,7 +170,7 @@ export function inverseCodeRange(ranges: CodeRange[]): CodeRange[] {
  * Find all code ranges that match the given pattern.
  *
  * @param code - The code to search in.
- * @param pattern - Either a string or a regular expression to search for.
+ * @param pattern - Either a string or a regular expression to search for. When a string is passed, it looks for **exact matches**. When a RegExp object is passed, it will match against it.
```

Comment:
Nit but can you apply this formatting to stay in the character limit:
```suggestion
 * @param pattern - Either a string or a regular expression to search for.
 *                  When a string is passed, it looks for **exact matches**.
 *                  When a RegExp object is passed, it will match against it.
```
Sadly I don't have a formatter configured for that yet.